src/test/frontend/db/query_dsl_test.cljs

```diff
@@ -1,10 +1,10 @@
 (ns frontend.db.query-dsl-test
-  (:require [cljs.test :refer [are async deftest testing use-fixtures]]
```

Comment:
New tests were added in this ns and property and page-property tests were refactored. New testing helpers were introduced to improve the accuracy of test assertions